packages/office-addin-dev-certs/src/verify.ts

```diff
@@ -25,7 +26,18 @@ export function isCaCertificateInstalled(): boolean {
     try {
         const output = execSync(command, {stdio : "pipe" }).toString();
         if (process.platform === "darwin") {
-            return true;
+            // Ceritificate exists. Now check and see if it's expired and remove it if is.
```

Comment:
We should fix this for Windows as well. Even in windows, isCaCertificateInstalled returns false if cert is expired, skipping uninstall phase.

Reply:
I validated that this is not an issue on Windows, but yes we should remove expired certs for both platforms

Reply:
@asachin96 I spoke with Louis and we decided to do the Windows expired cert check as a separate work item and confine this PR to just Mac so we can quickly get a fix out.  See VSO.3894960: Update office-addin-dev-certs to remove expired certs on Windows

Reply:
Alright